include/battle.h

```diff
@@ -901,6 +902,10 @@ struct MonSpritesGfx
     struct SpriteTemplate templates[4];
 };
 
+extern struct BattleSpritesGfx* gBattleSpritesGfx;
+extern u8 gBattleOutcome;
```

Comment:
I'd rather not have any externed variables in that header and for now just extern any needed variable. Then as it's more organized, move the variables to a specific's file header.